style/sass/style.scss

```diff
@@ -577,6 +580,131 @@ body {
     }
 }
 
+
+// pricing grid
+.pricing-grid {
+    display: grid;
+    grid-template-columns: repeat(3, 1fr);
+
+    &__plan {
+        border: 1px solid rgba(213, 174, 133, 0.57);
+        height: 512px;
+    }
+
+    &__plan:first-of-type {
+        border-top-left-radius: v.$radius;
+        border-bottom-left-radius: v.$radius;
+    }
+
+    &__plan:nth-of-type(2) {
+        height: 560px;
+        margin-top: -48px;
+        border-top-left-radius: v.$radius;
+        border-top-right-radius: v.$radius;
```

Comment:
można tutaj użyć 
`border-radius: v.$radius v.$radius 0 0;`

Zdecydowanie uszczupli to kod bo widzę że jest więcej takich przypadków